Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Base image containing dependencies used in builder and final image
-FROM swissgrc/azure-pipelines-dotnet:6.0.407 AS base
+FROM ghcr.io/swissgrc/azure-pipelines-dotnet:6.0.407 AS base
```

Comment:
Have you checked if Renovate is capable of handling updates from ghcr?

Reply:
I could not find anything specific in the docs,
but renovate itself has base images with ghcr.io registry (see https://github.com/renovatebot/docker-renovate-full/pull/1560)

I guess as long as the registry is public, no further steps are required.